df_py/volume/reward_calculator.py

```diff
@@ -176,6 +182,20 @@ def calc_rewards__usd(self) -> np.ndarray:
 
         # compute rewards
         R = np.zeros((N_i, N_j), dtype=float)
+
+        OCEAN_boost_limit_predictoor = 0.0
+        # Boost is limited to Sapphire mainnet only
+        if SAPPHIRE_MAINNET_CHAINID in self.predictoor_feed_addrs:
+            # Total boost is limited by PREDICTOOR_OCEAN_BUDGET
+            # Get the number of predictoor assets and calculate the limit per asset
+            n_predictoor_assets = len(
+                self.predictoor_feed_addrs[SAPPHIRE_MAINNET_CHAINID]
+            )
+            if n_predictoor_assets != 0:
+                OCEAN_boost_limit_predictoor = (
+                    PREDICTOOR_OCEAN_BUDGET / n_predictoor_assets
+                )
+
```

Comment:
You don't need any of the changes in reward_calculator.py.

Also, you need predictoor multiplier to be 0.201 for weeks before DF82.

Simplest is the following.

1. Remove `PREDICTOOR_MULTIPLIER` from constants.py, so you don't accidentally use it

2. Within `dcv_multiplier.py::calc_dcv_multiplier()`, change

From this:
```python
    if is_predictoor:
        return PREDICTOOR_MULTIPLIER
```

To this:
```python
    if is_predictoor:
        if DF_week < 82:
            return 0.201
        return 1.005
```

That's it! 

(And make sure you have a unit test for DF_week=81 and DF_week=82)

Reply:
No decision has changed. You are correct, it only applies to Predictoor.

And the proposed code supports this: that is why it has "if predictoor". 

There is still the non-predictoor section of code below the "if Predictoor" section. It doesn't change.